content/en/docs/reference/glossary/eviction.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Eviction
+id: eviction
+date: 2021-04-27
+full_link: /docs/concepts/scheduling-eviction/pod-eviction/
+short_description: >
+  Eviction is the process by which the kubelet proactively fails pods to reclaim
+  resources for nodes.
+aka:
+tags:
+- Operation
+- Fundamental
+---
+Eviction is the process by which the kubelet proactively fails pods to reclaim
```

Comment:
@dashpole this definition isn't correct, is it? From what you've said in your reviews, it isn't *just* the kubelet that evicts pods, the API does too? 

So would the definition be more correct as: "Eviction is the process by which pods are terminated proactively to reclaim resources for nodes", and then in the <!-- more --> section mention that eviction can happen automatically through kubelet or manually through the Eviction API?

Reply:
I would say "Out-of-resource Eviction is the process by which pods are terminated proactively to reclaim resources for nodes"

You can also say: "If you want to delete a pod, but respect policy controls during deletion, you can create an `Eviction` object, which will gracefully terminate your pod".

If you only want to talk about the former (out-of-resource evictions), then your definition is correct.  If you want to include the latter, you would need to make the language more general to the point where it wouldn't mean much to users.

Reply:
It might be a good idea to separate the two in the glossary - "Out-of-resource eviction" and "Eviction object". That way we can include both glossary terms in our concept topic and make a clearer distinction between the two.

Reply:
> eviction object

I'd call that an eviction resource. The difference is subtle but I'd be confident that creating an eviction means creating a resource, whereas different folks might or might not agree that the thing you create is an object.

Reply:
(the downside though is that “resource” is also the thing you can be short of when the kubelet evicts your Pod. That's, uh, a ~bug~ feature of English, though, and not in our power to adjust).

Reply:
:+1: for two glossary definitions. Have a think about what the short names for these should be.

Reply:
“graceful Pod deletion” could be the term for the thing you do when draining a node / creating a Pod eviction using the API